manifests/init.pp

```diff
@@ -160,6 +160,9 @@
 #
 # $cors_domains::                 List of domains that show be allowed for Cross-Origin Resource Sharing
 #
+# $trusted_proxies::              List of trusted IPs / networks. Default: localhost and all private IP address ranges.
```

Comment:
This is incorrect because Foreman overrides this to only localhost on v4 and v6:
https://github.com/theforeman/foreman/blob/a6471d0ac59ce83f6fa765da149e367aa6d459ec/config/application.rb#L200-L208

Also note that the way it works is that you must list **all** trusted IPs. That also means localhost if you override it.